public/pages/DetectorResults/containers/AnomalyHistory.tsx

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import React, { useState, useEffect, useCallback, Fragment } from 'react';
+import {
+  EuiFlexItem,
+  EuiFlexGroup,
+  EuiSpacer,
+  EuiTabs,
+  EuiTab,
+  EuiLoadingSpinner,
+} from '@elastic/eui';
+import moment from 'moment';
+import { useSelector, useDispatch } from 'react-redux';
+import { AppState } from 'public/redux/reducers';
+import {
+  AnomalyData,
+  Detector,
+  Monitor,
+  DateRange,
+} from '../../../models/interfaces';
+import {
+  getAnomalyResultsWithDateRange,
+  filterWithDateRange,
+} from '../../utils/anomalyResultUtils';
+import { get, isEmpty } from 'lodash';
+import { AnomalyResultsTable } from './AnomalyResultsTable';
+import { AnomaliesChart } from '../../AnomalyCharts/containers/AnomaliesChart';
+import { FeatureBreakDown } from '../../AnomalyCharts/containers/FeatureBreakDown';
+import { minuteDateFormatter } from '../../utils/helpers';
+import { ANOMALY_HISTORY_TABS } from '../utils/constants';
+
+interface AnomalyHistoryProps {
+  detectorId: string;
```

Comment:
minor: this might not be necessary as you can get detectorId by `detector.id`

Reply:
sure, will remove